compiler/compile/OMRSymbolReferenceTable.cpp

```diff
@@ -713,7 +717,7 @@ OMR::SymbolReferenceTable::findOrCreateCodeGenInlinedHelper(CommonNonhelperSymbo
    {
    if (!element(index))
       {
-      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_Helper);
+      TR::MethodSymbol * sym = TR::MethodSymbol::create(trHeapMemory(),TR_None);
```

Comment:
Correct. For helpers that require actual linkage, it may need TR_Helper, TR_System, TR_Private, or even other enum, depending on the implementation of the implementation of the helper. For example, TR_newObject needs TR_Helper,  TR_AMD64getTimeOfDay needs TR_System, TR_volatileReadLong needs TR_Private, etc.